### PR TITLE
Create Channel + Channel Dashboard + Chain Alias + Send notification feature

### PR DESCRIPTION
**Pull request by abhishek-01k**
## Pull Request Template

This contains PR number
- #1670
- #1629 
- #1704 
- #1724
- #1747
- #1752
- #1756
- #1795 
- #1796 
- #1729
- #1703

### Description

- **Problem/Feature**:

### Type of Change

- [ ] Bug fix
- [ ] New feature
- [ ] Code refactor
- [ ] Documentation update
- [ ] Other (please describe):

### Checklist

- [ ] **Quick PR**: Is this a quick PR? Can be approved before finishing a coffee.
  - [ ] Quick PR label added
- [ ] **Not Merge Ready**: Is this PR dependent on some other PR/tasks and not ready to be merged right now.
  - [ ] DO NOT Merge PR label added

### Frontend Guidelines

- [ ] Followed frontend guidelines as per the [Guidelines Notion Doc](https://www.notion.so/pushprotocol/Frontend-dApp-Guidelines-1d7806ae3d9e4569a340b563dcd0536c)

### Build & Testing

- [ ] No errors in the build terminal
- [ ] Engineer has tested the changes on their local environment
- [ ] Engineer has tested the changes on deploy preview

### Screenshots/Video with Explanation

- **Before:** Explain the previous behavior

- **After:** What's changed now

### Review & Approvals

- [ ] Self-review completed
- [ ] Code review by at least one other engineer
- [ ] Documentation updates if applicable

### CODE REVIEW

**Comment by rohitmalhotra1420** [comment]
@mishramonalisha76 
- The send notification option is not visible every time.
- After I add the new chain alias I am redirected to the older dashboard page. It should be the new /channel/id route.
- I cannot see the chain alias after I have added it.

**Comment by rohitmalhotra1420** [comment]
Screenshot 2024-07-31 at 3 56 36 PM
Let's add spacing here between the colon and the number @abhishek-01k

**Review by mishramonalisha76 — CHANGES_REQUESTED** [review]
if the alias is not verified, the channel should show a loading page . For now it shows the channel detials.
https://www.figma.com/design/FvJL0JWHlsVohsHTERB1Lh/Push-Design-Foundations?node-id=6822-8391&m=dev

**Comment by mishramonalisha76** [comment]
https://github.com/user-attachments/assets/37b44796-2962-4ccf-b6ff-7efc5714ecd9

@rohitmalhotra1420  I have pushed a fix for the select popover to disappear when we scroll in the parent.

File: src/blocks/select/Select.tsx [hunk 1]
```diff
@@ -219,19 +219,10 @@ const Select: React.FC<SelectProps> = ({
...
   const childRef = useRef<HTMLInputElement>(null);
 
-  // useEffect(() => {
-  //   const handleResize = () => {
-  //     if (comboboxRef.current) {
-  //       setPopoverWidth(comboboxRef.current.offsetWidth - 18);
-  //     }
-  //   };
+  useEffect(() => {
+    window.addEventListener('scroll', () => setViewPopover(false));
```
Comment (rohitmalhotra1420) [inline src/blocks/select/Select.tsx:223]: @mishramonalisha76 we also need to remove the listener on component unmount.

**Review by rohitmalhotra1420 — CHANGES_REQUESTED** [review]

File: src/blocks/select/Select.tsx (hunk 1 above)
Comment (mishramonalisha76) [reply src/blocks/select/Select.tsx]: done

**Comment by rohitmalhotra1420** [comment]
@mishramonalisha76 I guess we should a warning message to the user before creating a new chain that they do not interupt the flow while alias creation.

**Comment by mishramonalisha76** [comment]
@abhishek-01k 
- Polling isnt working after adding alias
- Alais should not see the add new chain plus icon in the channel dashboard 
- Alais should not be able to update settings, disable the manage settings button.
-

**Comment by rohitmalhotra1420** [comment]
@mishramonalisha76 we need to fix the polling after chain alias creation.

**Comment by mishramonalisha76** [comment]
@abhishek-01k  
image
 the hover borderRadius is bot correct, also On mobile view i am unable to see the channel name

**Review by rohitmalhotra1420 — APPROVED** [review]

**Pull request merged**